src/makeRobot.js

```diff
@@ -38,7 +38,69 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    x: 0,
+    y: 0,
```

Comment:
As far as I can remember, these should be stored in coords object)

```suggestion
    coords: {
      x: 0, y: 0,
    },
```